test/unit/keypair_test.js

```diff
@@ -1,52 +1,64 @@
-
 describe('Keypair.contructor', function() {
-  it("fails when passes secret key does not match public key", function() {
-    let secret = "SD7X7LEHBNMUIKQGKPARG5TDJNBHKC346OUARHGZL5ITC6IJPXHILY36";
+  it('fails when passes secret key does not match public key', function() {
```

Comment:
oh, this is replaced automatically by some linter on this repo -- ignore this changes